plop-templates/component/src/react/index.tsx.hbs

```diff
@@ -1,40 +1,36 @@
-import { StyleAttribute, compose, css } from 'glamor'
-import React, { HTMLAttributes } from 'react'
-
-import { useTheme } from '@pluralsight/ps-design-system-theme'
-import filterReactProps from '@pluralsight/ps-design-system-filter-react-props'
+import { ValueOf } from '@pluralsight/ps-design-system-util'
+import { names as themeNames, useTheme } from '@pluralsight/ps-design-system-theme'
+import { compose, css } from 'glamor'
+import React, { forwardRef } from 'react'
 
 import stylesheet from '../css'
 
-interface {{pascalCase name}}Props extends extends HTMLAttributes<HTMLDivElement> {}
-
-type StyleFn = (themeName:string, props: {{pascalCase name}}Props) => StyleAttribute
+type DivEl = JSX.IntrinsicElements['div']
+interface {{pascalCase name}}Props extends DivEl {}
```

Comment:
have we settled on using an alias?

Reply:
Dunno if we've settled on that, but it is the latest practice.  Seems like we've settled on moving away from HTMLAttributes.

Should we settle?

If so, on PropsOf or alias?